### PR TITLE
Adding Managed Instance Dashboard v0.4.1.

### PR DESCRIPTION
**Pull request by jovanpop-msft**
This PR fixes:
- Packaging issues (removed unnesecary files)
- Collation issues
- Changed icons.

> NOTE: I'm the author of this extension and I'm pushing new version.

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -1622,14 +1622,14 @@
...
-							"lastUpdated": "11/5/2019",
+							"version": "0.4.1",
+							"lastUpdated": "03/08/2020",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/managed-instance-dashboard/managed-instance-dashboard-0.3.9.vsix"
+									"source": "https://github.com/JocaPC/AzureDataStudio-Managed-Instance/releases/download/0.4.1/managed-instance-dashboard-0.4.1.vsix"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json]: For direct VSIX installs we use the blob storage account. I'll handle uploading the file - just change the URL to `https://sqlopsextensions.blob.core.windows.net/extensions/managed-instance-dashboard/managed-instance-dashboard-0.4.1.vsix`

Comment (jovanpop-msft) [reply extensionsGallery.json]: I changed the path. File is in releases: https://github.com/JocaPC/AzureDataStudio-Managed-Instance/releases/download/0.4.1/managed-instance-dashboard-0.4.1.vsix

**Comment by alanrenmsft** [comment]
@jovanpop-msft please also update the insider gallery.

**Comment by jovanpop-msft** [comment]
@alanrenmsft @Charles-Gagnon I added all changes.

**Pull request merged**